lib/msal-node/src/client/PublicClientApplication.ts

```diff
@@ -187,9 +199,12 @@ export class PublicClientApplication
 
             const authCodeUrl = await this.getAuthCodeUrl(validRequest);
             await openBrowser(authCodeUrl);
-            const authCodeResponse = await authCodeListener.finally(() => {
+            await authCodeListener.finally(() => {
```

Comment:
if you change the try/catch to a try/finally you can avoid needing to call `closeServer` in two places.

Reply:
Done. Thanks for the suggestion!